package.json

```diff
@@ -78,6 +79,7 @@
     "@testing-library/react-hooks": "3.2.1",
     "@types/jest": "24.0.23",
     "@types/react": "16.9.11",
+    "@xobotyi/scrollbar-width": "1.5.0",
```

Comment:
We already have it in `"dependencies"`, probably this line is not needed.

Reply:
Forgot to remove from dev-deps.